### PR TITLE
Improving translation for PT

### PR DESCRIPTION
**Pull request by guiananias**
Hello!
I made some improvements regarding the translation into Portuguese, among them:

- Translation correction;
- Ortography correction;

### CODE REVIEW

File: po/pt.po [hunk 1]
```diff
@@ -182,11 +182,11 @@ msgstr "Diffs a mostrar?"
 
 #: clean.go:89
 msgid "Do you want to remove ALL AUR packages from cache?"
-msgstr "Quer remover todos os pacotes AUR da cache?"
+msgstr "Quer remover todos os pacotes AUR do cache?"
```
Comment (Jguer) [inline po/pt.po:185]: I follow the priberam dictionary on this so cache should be feminine

https://dicionario.priberam.org/cache

Comment (guiananias) [reply po/pt.po:185]: Yes Yes. If it is the case to follow the dictionary, cache is a feminine word, you are right.

Is it that, generally speaking, we tend to bring the "cache" to the male? Now why is a good question. :joy:

I will make the correction, thank you.

**Comment by agendator** [comment]
Hi! I'd like to point out that these changes made by @guiananias are meant for a Brazilian Portuguese translation, which is not the case. This input will break what was meant to be a Portugal Portuguese translation. The best solution is to make a separate ptBR translation, since ptPT and ptBR have so many differences.

**Comment by guiananias** [comment]
@agendator yeah, good point i forgot to mention. 

Indeed, in this case, the use of Portuguese is actually located in Brazilian Portuguese, which, although not different in writing, are languages that are quite different.

**Comment by Jguer** [comment]
I use `pt_PT` and I can understand these fixes to the translation ( crear and analizar :man_facepalming: ). 

The most shocking differences for me is `usuario` versus `utilizador` and `baixado` vs `descarregado` but even for the `pt_PT` translations I would keep some of these changes.

I propose we do like [pacman](https://git.archlinux.org/pacman.git/tree/src/pacman/po) and use `pt.po` for pt_PT and `pt_BR.po` for pt_BR.

On this MR:
- Keep the changes to the `pt.po` as they are, I'll cherrypick the most appropriate
- Add a new commit with a `pt_BR.po` (don't forget to add pt_BR to the makefile `LANGS`) 

Would that work for you @guiananias ?

**Comment by guiananias** [comment]
@Jguer it's fine by me. I will add the new file.

**Pull request merged**